### ISSUE TITLE
New output channel: Assign custom variables

### ISSUE THREAD

**Issue by Taitava** [Taitava/obsidian-shellcommands#382]
### Discussed in https://github.com/Taitava/obsidian-shellcommands/discussions/127

This output channel assumes the output is a JSON object, reading it's properties and assigning them to custom variables.

For example, consider the following JSON output:
```JSON
{
    "_link_to_file": "My folder/My note.md",
    "_link_alias": "Meeting notes"
}
```

This would assign the value `My folder/My note.md` to a custom variable named `{{_link_to_file}}`, and `Meeting notes` to `{{_link_alias}}`.

TODO:
- [x] Implement the thing.
- [x] Add to CHANGELOG.md
- [x] Write documentation.

**Comment by Taitava** [comment]
Done. Will be released in `0.22.0`.

**Comment by Taitava** [comment]
This is now released.
